src/pyclaw/sharpclaw/flux1.f90

```diff
@@ -43,13 +43,15 @@ subroutine flux1(q1d,dq1d,aux,dt,cfl,t,ixyz,num_aux,num_eqn,mx,num_ghost,maxnx,r
     double precision, target, intent(in) :: aux(num_aux,1-num_ghost:mx+num_ghost)
     double precision, intent(out) :: cfl
     double precision, intent(in) :: t, dt
+    logical, intent(in) :: downwind
```

Comment:
Actually I think it is great that you put it here.  I may move some things out of ClawParams in the future so its more obvious where they are used.